### ISSUE TITLE
What does grad_norm stand for?

### ISSUE THREAD

**Issue by chenhao2015** [zhyever/Monocular-Depth-Estimation-Toolbox#75]
Hi @zhyever, thanks for your great work.

Can I ask what grad_norm (Please see the picture) stands for? I checked the code and didn't find the corresponding code location.
![image](https://user-images.githubusercontent.com/13285296/212712700-319b50c0-bd4b-41ef-be10-f57746f9f296.png)

**Comment by chenhao2015** [comment]
> Thanks! Hope this repo is helpful for you!
> 
> Please refer to [codes](https://mmcv.readthedocs.io/en/latest/_modules/mmcv/runner/hooks/optimizer.html).
> 
> ```
> if grad_norm is not None:
>     # Add grad norm to the logger
>     runner.log_buffer.update({'grad_norm': float(grad_norm)},
>     runner.outputs['num_samples'])
> ```
> 
> The program will automatically log the `grad_norm`(gradient norm) to indicate the fluctuation and stability of the training process.

Thanks for your answering.

**Comment by zhyever** [comment]
Close for now.
